src/GeneralStateTestsFiller/VMTests/vmTests/push0Filler.yml

```diff
@@ -0,0 +1,185 @@
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f5f\
+        5f"
+      nonce: '0'
+      storage: {}
+
+  transaction:
+    data:
+      - ':label single_push0 :raw 0x0000000000000000000000000000000000000100'
+      - ':label 1024_push0   :raw 0x0000000000000000000000000000000000000200'
+      - ':label 1025_push0   :raw 0x0000000000000000000000000000000000000300'
+    gasLimit:
+      - 700000
+    gasPrice: 10
+    nonce: 0
+    secretKey: 45a915e4d060149eb4365960e6a7a45f334393093061116b197e3240065ff2d8
+    to: 'b94f5374fce5edbc8e2a8697c15331677e6ebf0b'
+    value:
+    - 0
+        
+  expect:
+    - indexes:
+        data: ':label single_push0'
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - 'London'
+      result:
+        0000000000000000000000000000000000000100:
+          storage: {}
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+            0x01: 0x01
+    - indexes:
+        data: ':label single_push0'
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - 'London+3855'
+      result:
+        0000000000000000000000000000000000000100:
+          storage:
+            0x00: 0x01
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+            0x00: 0x01
+            0x01: 0x01
+
+    - indexes:
+        data: ':label 1024_push0'
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - 'London'
+      result:
+        0000000000000000000000000000000000000200:
+          storage: {}
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+            0x01: 0x01
+    - indexes:
+        data: ':label 1024_push0'
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - 'London+3855'
+      result:
+        0000000000000000000000000000000000000200:
+          storage:
+            0x00: 0x01
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+            0x00: 0x01
+            0x01: 0x01
+
+    - indexes:
+        data: ':label 1025_push0'
+        gas:  !!int -1
+        value: !!int -1
+      network:
+        - 'London'
+      result:
+        b94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          storage:
+            0x01: 0x01
+    # Same expectation, but for London execution fails on 5f opcode, for 3855 it fails with stack overflow     
```

Comment:
new opcode to be added into opcode tests by Ori. 
https://github.com/ethereum/tests/tree/develop/src/GeneralStateTestsFiller/stBadOpcode
and here https://github.com/ethereum/tests/tree/develop/src/Templates/BadOpcode

ask Ori to add this opcode once the realisation is certain